atomic_reactor/tasks/common.py

```diff
@@ -107,3 +112,31 @@ def load_workflow_data(self) -> inner.ImageBuildWorkflowData:
 
     def load_config(self) -> config.Configuration:
         return config.Configuration(self._params.config_file)
+
+    def throw_canceled_build_exception(self):
+        self.data.build_canceled = True
+        raise BuildCanceledException("Build was canceled")
+
+    @classmethod
+    def run(cls, init_build_dirs: Optional[bool] = False):
+        try:
+            cls.fs_watcher.start()
```

Comment:
I don't think this should be a class method, fs_watcher is an instance attribute (and execute() and others as well)